config/go.d/web_log.conf

```
@@ -218,3 +218,11 @@ jobs:
 
   - name: gunicorn
     path: /var/log/gunicorn/gunicorn-access.log
+
+  # IIS
+  # This configuration assumes you are running netdata on WSL
+  #- name: iis
+  #  path: /mnt/c/inetpub/logs/LogFiles/W3SVC1/u_exYYMMDD.log
```

```suggestion
  #  path: /mnt/c/inetpub/logs/LogFiles/W3SVC1/u_exYYMMDD.log
```

Should use `*` instead of YYMMDD

---

Done